src/Microsoft.Diagnostics.Monitoring.Options/OptionsDisplayStrings.resx

```diff
@@ -551,4 +551,12 @@
   <data name="DisplayAttributeDescription_GlobalCounterOptions_IntervalSeconds" xml:space="preserve">
     <value>Determines the metrics interval for all dotnet-monitor scenarios. This includes prometheus metrics, live metrics, triggers, and traces.</value>
   </data>
+  <data name="DisplayAttributeDescription_LoadProfilerOptions_Path" xml:space="preserve">
+    <value>Defines the path to the dll for the profiler to be loaded. This is typically the same value that would be passed in the environment variable CORECLR_PROFILER_PATH.</value>
```

Comment:
```suggestion
    <value>The path of the profiler library to be loaded. This is typically the same value that would be set as the CORECLR_PROFILER_PATH environment variable.</value>
```

Let's avoid saying "dll" because it's typically a `.dll` on Windows, `.so` on Linux, and `.dylib` on MacOS.